NYCFireWire/NYCFireWire/src/Views/IncidentTableViewCell.swift

```diff
@@ -8,17 +8,27 @@
 
 import UIKit
 
-class IncidentTableViewCell: UITableViewCell {
+protocol LikeButtonDelegate {
+    func incidentLikedTapped(cell: IncidentTableViewCell)
+}
 
+class IncidentTableViewCell: UITableViewCell {
     
     @IBOutlet weak var boroLabel: UILabel!
     @IBOutlet weak var boxLabel: UILabel!
     
     @IBOutlet weak var titleLabel: UILabel!
     @IBOutlet weak var subtitleLabel: UILabel!
     
-    @IBOutlet weak var addressLabel: UILabel!
-    @IBOutlet weak var timeLabel: UILabel!
+    @IBOutlet weak var dateLabel: UILabel!
+    
+    @IBOutlet weak var likeButton: UIButton!
+    @IBOutlet weak var numberOfLikesLabel: UILabel!
+    
+    @IBOutlet weak var numberOfCommentsLabel: UILabel!
+    
+    var isLiked: Bool?
```

Comment:
1. Make sure we set this in cellForRowAtIndexPath
2. When this is set, we have to make sure to update the button image

Reply:
Also, instead of making this optional, just set it to false initially